src/app/home/home.page.html

```diff
@@ -1,8 +1,65 @@
 <app-application-header [config]="headerConfig" (actionClick)="handleActionClick($event)"></app-application-header>
 <ion-content [fullscreen]="true">
-  <!-- <app-carousel [items]="carouselItems" [template]="itemTemplate" [itemsToShow]="4"></app-carousel>
+  <div class="ion-margin-vertical">
+    <app-section-header [headerName]="'My Improvements'" [showViewmore]="myImprovements.length > 0" [navigateUrl]="'/my-improvements-listing'"></app-section-header>
```

Comment:
why are we checking myImprovements.length > 0 this for show view more? shouldn't that come if data count is more than the number of items shown inside carousel?

Reply:
if there are no improvements at all we have to don't need to show the view more